db/migrate/20160412102905_create_container_deployment_custom_settings.rb

```diff
@@ -0,0 +1,10 @@
+class CreateContainerDeploymentCustomSettings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployment_custom_settings do |t|
+      t.string :path
+      t.text :value
+      t.references :container_deployment, :index => {:name => 'deployment_settings'}
```

Comment:
[Same thing here](https://github.com/ManageIQ/manageiq/pull/7779/files#r59564140) ...

`t.belongs_to` and `bigint`